webmention.php

```diff
@@ -63,6 +63,9 @@ function init() {
 	require_once dirname( __FILE__ ) . '/includes/class-comment.php';
 	add_action( 'init', array( '\Webmention\Comment', 'init' ) );
 
+	require_once dirname( __FILE__ ) . '/includes/class-comment-walker.php';
```

Comment:
we should add an option/const/add_theme_support to disable the "comment representation" feature of the Webmention plugin.

Reply:
I don't think it is necessary at the place you identify. As of now, the Comment Walker just adds additional options to wp_list_comments. The default is supposed to be false...noticed I committed it as true by accident. 

The options for how to display differently will likely be changing the default display to exclude the mention types, and allowing them to be called separately, either by using the hook we use in Semantic Linkbacks, or the theme calling wp_list_comments for them specifically.